### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by leo-liu1**

### CODE REVIEW

**Comment by leo-liu1** [comment]
I recommend pulling from this branch and running it on local machine to see how it looks

**Comment by leo-liu1** [comment]
Also, I just noticed that for some reason the transparent opacity on the landing page makes the mobile links hard to click. Especially the ones towards the bottom.

**Comment by vfcheung** [comment]
Since the landing section is now a static blue color, we don't need a transparent navbar anymore right? This means we can make it so the navbar no longer overlaps with the landing section, and so the links towards the bottom can actually be clicked on mobile.

File: src/app/Components/ContactButton.js [hunk 1]
```diff
@@ -4,27 +4,28 @@ import { Col, Container, Row, Button } from 'react-bootstrap';
 
 export default function ContactUs() {
   return (
-    <div class="container-fluid py-3" style={contactStyle}>
+    <div className="container-fluid py-3" style={contactStyle}>
       <Container>
-      <Row>
+        <Row>
+          <Col xs={{ span: 10, offset: 1 }} md={{ span: 5, offset: 0}}>
```
Comment (vfcheung) [inline src/app/Components/ContactButton.js]: Do we want the content to be centered on medium screens? Maybe md={{ span: 6, offset: 3 }} or something?

**Pull request merged**